### PR TITLE
Add pre-commit. Fix flake8 linting issues

### LINKED ISSUE #266: Add Git pre-commit hooks
For Black formatting and Flake8 (#265), as discussed in #262.

We can use https://pre-commit.com/ or some other tool.

### PR DESCRIPTION
**Pull request by JoepdeJong**
closes #266

### CODE REVIEW

File: ldb/export.py [hunk 1]
```diff
@@ -294,7 +295,7 @@ def getname(obj):
                     name = obj.get("organization__name_prefix", "")
                     name += " "
                     name += obj.get("organization__name")
-                    name = re.sub("\s+", " ", name)
+                    name = re.sub(" +", " ", name)
```
Comment (praseodym) [inline ldb/export.py:298]: Why is this changed? `\s` matches more than just spaces.

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "wisvch-dienst2"
+version = "1.13"
```
Comment (praseodym) [inline pyproject.toml:7]: Where does this version come from? I personally don't think we should do any versioning.

File: .pre-commit-config.yaml
Comment (praseodym) [inline .pre-commit-config.yaml:1]: Good idea to add pre-commit, but we should:
- check this in GitHub Actions
- add instructions to README.md on how to set up pre-commit hook (`pre-commit install`)
- install pre-commit in GitHub Codespaces

File: ldb/export.py (hunk 1 above)
Comment (JoepdeJong) [reply ldb/export.py:298]: Flake8 says:
```
./ldb/export.py:298:36: W605 invalid escape sequence '\s'
./ldb/export.py:320:36: W605 invalid escape sequence '\s'
```

Comment (praseodym) [reply ldb/export.py:298]: Hm it's not a escape sequence but part of the regex, maybe it should be `\\s+` instead?

Comment (jgadelange) [reply ldb/export.py:298]: I think it might need to be `r"\s+"`

File: .pre-commit-config.yaml
Comment (JoepdeJong) [reply .pre-commit-config.yaml:1]: why do we need to check pre-commit in GH actions? the actions already check `black` and `flake8`

Comment (praseodym) [reply .pre-commit-config.yaml:1]: I'd prefer just checking pre-commit and skipping explicit actions to run `black` and `flake8`. This way it's easier to add more linters to pre-commit (e.g. [Ruff](https://beta.ruff.rs/docs/)) and keep versions in sync because we only need to maintain the pre-commit settings.

**Review by praseodym — APPROVED** [review]
